src/super_gradients/recipes/dataset_params/coco_pose_estimation_dataset_params.yaml

```diff
@@ -0,0 +1,84 @@
+train_dataset_params:
+  data_dir: /data/coco # root path to coco data
+  images_dir: images/train2017
+  json_file: annotations/person_keypoints_train2017.json
+
+  include_empty_samples: False
+  min_instance_area: 128
+
+  transforms:
+    - KeypointsLongestMaxSize:
+        max_sizes: [ 640, 640 ]
+
+    - KeypointsPadIfNeeded:
+        output_size: [ 640, 640 ]
+        image_pad_value: [ 127, 127, 127 ]
+        mask_pad_value: 1
+
+    - KeypointsRandomHorizontalFlip:
+        # Note these indexes are COCO-specific. If you're using a different dataset, you'll need to change these accordingly.
+        flip_index: [ 0, 2, 1, 4, 3, 6, 5, 8, 7, 10, 9, 12, 11, 14, 13, 16, 15 ]
+        prob: 0.5
+
+    - KeypointsRandomAffineTransform:
+        input_size: [ 640, 640 ]
+        output_size: [ 640, 640 ]
+        max_rotation: 30
+        min_scale: 0.75
+        max_scale: 1.5
+        scale_type: 'short'
+        max_translate: 40
+        image_pad_value: [ 127, 127, 127 ]
+        mask_pad_value: 1
+
+    - KeypointsToTensor: {}
+
+    - KeypointsNormalize:
+        mean: [ 0.485, 0.456, 0.406 ]
+        std: [ 0.229, 0.224, 0.225 ]
+
+  target_generator: ???
+
+val_dataset_params:
+  data_dir: /data/coco/
+
+  images_dir: images/val2017
+  json_file: annotations/person_keypoints_val2017.json
+  include_empty_samples: True
+  min_instance_area: 128
+  transforms:
+    - KeypointsLongestMaxSize:
+        max_sizes: [ 640, 640 ]
+
+    - KeypointsPadIfNeeded:
+        output_size: [ 640, 640 ]
+        image_pad_value: [ 127, 127, 127 ]
+        mask_pad_value: 1
+
+    - KeypointsToTensor: {}
```

Comment:
no need for ": {}"

Reply:
Ah, I see.
If I type just `- KeypointsToTensor` it's instantiated correct, but If I use `- KeypointsToTensor: ` then it fails inside base factory `return self.type_dict[_type](**_params)` since `_params` is None.

Do you think it make sense to handle this as well like so:
`return self.type_dict[_type](**_params or {})` ?

Reply:
but shouldn't it fail anyways in that case since None is not unpackable ?

Reply:
Think we can leave it as is for now...